projects/js-themes-toolkit/packages/liferay-theme-tasks/lib/project/options.js

```diff
@@ -87,7 +91,7 @@ class Options {
 				Option ${optionName} contains '\\' characters.
 
 				This can be due to using Windows paths in a config file, which is not
-				supported, or because its use is legitimate in POSIX paths. 
+				supported, or because its use is legitimate in POSIX paths.
```

Comment:
:ok_hand:

Reply:
This is probably the most important contribution of the PR. 

The rest is: not horrible...